config/config.yaml

```diff
@@ -0,0 +1,123 @@
+# This file contains defaults for the "config" object used in the Snakefile.
+# To temporarily override or provide a value, you can use snakemake's --config
+# or --configfile options.
+---
+s3_staging_url: s3://nextstrain-staging
```

Comment:
```suggestion

# S3 and slack options are related to Nextstrain live builds and don't need to be modified for local builds
s3_staging_url: s3://nextstrain-staging
```

I know this seems a little silly but if something's not working people zero in on this kind of stuff amazingly quickly... Anything to help convince people that it's definitely not a problem with the slack-channel name is helpful! :)